app/src/newdocs/content/canvasModules/AddToList-548.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable quotes, quote-props, indent, comma-dangle, max-len */
+import moduleDescription from './AddToList-548.mdx'
+
+export default {
+  "id": 548,
+  "name": "AddToList",
+  "path": "List",
+  "help": {
+    "params": {
+      "index": "index to add to, from 0 to length of list"
+    },
+    "paramNames": [
+      "index"
+    ],
+    "inputs": {
+      "item": "item to add to list",
+      "list": "the list to add to"
```

Comment:
Also wondering if it'd be easier to auto-fix all of these to at least start with a capital letter using your importing script, rather than doing it later?